extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -109,3 +109,49 @@ tags: {  # amp-autocomplete JSON
     }
   }
 }
+# The template element is also defined at
+# extensions/amp-mustache/validator-amp-mustache.protoascii
+tags: {  # <amp-autocomplete> > <template>
+  html_format: AMP
+  tag_name: "TEMPLATE"
+  spec_name: "AMP-AUTOCOMPLETE > template"
+  mandatory_parent: "AMP-AUTOCOMPLETE"
+  requires_extension:"amp-mustache"
+  attrs: {
+    name: "type"
+    mandatory: true
+    value: "amp-mustache"
+    dispatch_key: NAME_VALUE_PARENT_DISPATCH
+  }
+  reference_points: {
+    tag_spec_name: "AMP-AUTOCOMPLETE > template > child"
+  }
+  reference_points: {
+    tag_spec_name: "AMP-AUTOCOMPLETE > template > children"
+  }
+}
+tags: {
+  html_format: AMP
+  tag_name: "$REFERENCE_POINT"
+  spec_name: "AMP-AUTOCOMPLETE > template > child"
+  attrs: {
+    name: "disabled"
+    value: ""
+  }
+  attrs: {
+    name: "value"
+    value: ""
```

Comment:
this means `value` can never have a value, it can only be `<foo value>`, `<foo value="">` or `<foo value=value>`. However in the test case it does have a value, which makes me wonder if this is working as intended.

Reply:
Hmm, we do want this to have a value, so I'll remove the `value: ""`. Wonder why the validator did not catch this, though.